redis/redisLimiter_test.go

```diff
@@ -389,6 +385,126 @@ func TestReset(t *testing.T) {
 	})
 }
 
+func TestSetSecurityModeNoExpire(t *testing.T) {
+	t.Parallel()
+
+	args := createMockRateLimiterArgs()
+
+	t.Run("should work", func(t *testing.T) {
+		t.Parallel()
+
```

Comment:
use a new args variable in each inner test, otherwise there might be some race conditions in tests

Reply:
updated